cpp/src/arrow/flight/sql/odbc/flight_sql/address_info.h

```diff
@@ -19,8 +19,10 @@
 
 #include <string>
 
-#include <arrow/flight/sql/odbc/odbcabstraction/include/odbcabstraction/platform.h>
+#include "arrow/flight/sql/odbc/odbcabstraction/include/odbcabstraction/platform.h"
+
 #include <sys/types.h>
+#include <cstdint>
```

Comment:
Can we keep headers sorted? standard library headers should go together (and go above project header includes)